### ISSUE TITLE
[packagers] Should the homebrew tap formula file name have a different casing than the cask name?

### ISSUE THREAD

**Issue by crschnick** [jreleaser/jreleaser#1652]
I'm currently creating another homebrew cask with a hyphen in its name. I'm not a homebrew expert, but I was under the impression that the formula file name should be equal to the formula/cask name in the file. But it seems like jreleaser modifies the .rb file name to remove hyphens and capitalize parts.

When creating a cask with a hyphen in jreleaser, the homebrew client will complain about the mismatch in file names and cask names and refuse to install the cask.

**Comment by aalmiray** [comment]
From the docs at https://docs.brew.sh/Cask-Cookbook

```
The Cask name (<cask-token>) on the header line cask <cask-token> do should match the cask filename, without the .rb extension, enclosed in double quotes.
```

**Comment by crschnick** [comment]
Alright then this is probably jreleaser bug. The file name is then not correct for casks, haven't tested with formulas yet.

**Comment by aalmiray** [comment]
Additional info regarding formula naming conventions https://docs.brew.sh/Formula-Cookbook#a-quick-word-on-naming

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.13.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.13.0))
